test/test_owners.csv

```diff
@@ -601,7 +601,7 @@ k8s.io/kubernetes/pkg/kubelet/dockertools,deads2k,1
 k8s.io/kubernetes/pkg/kubelet/envvars,karlkfi,1
 k8s.io/kubernetes/pkg/kubelet/eviction,childsb,1
 k8s.io/kubernetes/pkg/kubelet/images,caesarxuchao,1
-k8s.io/kubernetes/pkg/kubelet/lifecycle,david-mcmahon,1
+k8s.io/kubernetes/pkg/kubelet/lifecycle,caesarxuchao,1
```

Comment:
Perhaps should be owned by @yujuhong or other members in the node team?

Do you know how this file is generated? I see my name in a few places and they don't make sense.